src/app/shared/modules/filesystem/file-list-view/components/folder-list-item/folder-list-item.component.html

```diff
@@ -10,6 +10,13 @@
     {{ timestampAge }}
   </div>
   <div class="empty"></div>
+  <div class="toggleBundle" *ngIf="checkAccessGroups() else notAdminOrCurator">
+    <clark-toggle-switch 
+      [state]="getFolderBundleStatus(folder)" 
+      (click)="$event.stopPropagation()" 
+      (toggled)="handleBundleToggle(folder, $event)"
+      ></clark-toggle-switch>
+  </div>
```

Comment:
Not sure what needs to be refactored: let's just toggle this functionality completely off for authors. So this will only display if the user is only either a admin or curator